src/components/Todo.tsx

```diff
@@ -27,10 +27,24 @@ function Todo(props: { fetchList: any; addTodo: any }) {
     setTodo(event.target.value);
   };
 
-  const handleClick = (event: any) => {
+  const [newdata, setNewdata] = useState(null);
```

Comment:
I've noticed the other states are grouped together. Is there a reason this one is not part of that group? Btw, this doesn't require a change unless others would want it. I'm just pointing it out for consistency purposes.

Reply:
Right, I put this here because it's only used use for the `handleClick` function, while the other states on the top are used more extensively throughout the code.